Main.java

```diff
@@ -9,7 +11,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // create a scanner for user import
+    Scanner input = new Scanner(System.in);
+
+    // declare a variable to see if a number is divisible by two
+    final int DIVISIBLE_TWO = 2;
+
+    // find out user number
+    System.out.println("Please enter a number");
+    int number = input.nextInt();
+
+    // declare a variable for the divided number
+    int number2 = number%DIVISIBLE_TWO;
+
+    //figure out if number is divisible by two
+    if (number/DIVISIBLE_TWO == number2){
+      System.out.println("Your number is odd");
+    } else {
+      System.out.println ("Your number is even");
+    }
```

Comment:
So this logic is not quite correct. number2 is really the remainder when you divide by 2. The if statement should be if this remainder is 0 or not 0. That would tell you if it is even or odd.